scripts/parse-csv.py

```diff
@@ -0,0 +1,163 @@
+#! /usr/bin/env python
+#######################################################################
+# The script to prepare ALL of the "orbitals" data e.g. Sun, satellites
+#######################################################################
+
+import argparse
+import yaml
+import csv
+import h5py
+
+from datetime import datetime
+
+import numpy as np
+
+# lusee/opsim
+from    nav.coordinates import *
+from    lunarsky.time   import Time
+
+parse_time_string = '%d %b %Y %H:%M:%S.%f'
+# ----------------------------------------------------------------------------------
+parser = argparse.ArgumentParser()
+
+parser.add_argument("-v", "--verbose",      action='store_true', help="Verbose mode")
+parser.add_argument("-I", "--inspect",      action='store_true', help="Inspect CSV and exit")
+parser.add_argument("-c", "--conffile",     type=str,            help="Optional - a YAML file containing configuration", default='')
+parser.add_argument("-o", "--outputfile",   type=str,            help="The output", default='')
+parser.add_argument("-i", "--inputfile",    type=str,            help="The CSV file to read", default='')
+parser.add_argument("-f", "--fields",       type=str,            help="Column numbers to process, comma-separated, default ALL, zero is always included", default='')
```

Comment:
Not strictly necessary, but it might be better to refer to column by their name stored in the CSV header and have default time, and 3 MoonBody Cols. This way is more robust against how their future files might look like.